### PR TITLE
feat(project): installed bundles and setup starting parameters + UML diagrams

### PR DESCRIPTION
**Pull request by WizBhoo**
Hello Rui,
Could you please review this feature ? Thanks in advance.

**Purpose** : it's the first PR to init the project with Symfony, install bundles i will need as NelmioApiDocBundle and FOSRestBundle.
Moreover, this PR cover a part of my project analysis using UML to design my DataBase.
UML diagram are linked to this PR.

**Link to my project issues** : #2 / #5 

1_UseCase_Gestion_Catalogue_Produits_BileMo
2_Sequence_Gestion_Catalogue_Produits_BileMo
3_UseCase_Gestion_Utilisateurs
4-1_Sequence_Gestion_Utilisateurs_GET
4-2_Sequence_Gestion_Utilisateurs_POST_DELETE
5_UseCase_Authentification_JWT
6_Sequence_Authentification_JWT
7_BileMoAPIClassDiagram
8_MPD_BileMoAPI

### CODE REVIEW

File: config/packages/fos_rest.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    param_fetcher_listener:  true
+    allowed_methods_listener:  true
+    routing_loader: true
+    body_converter:
+      enabled: true
+    serializer:
+      serialize_null: true
+    view:
+        view_response_listener:  true
+#    exception:
```
Comment (ruiteix) [inline config/packages/fos_rest.yaml:12]: please remove dead code

File: composer.json [hunk 2]
```diff
@@ -5,6 +5,8 @@
         "php": "^7.1.3",
```
Comment (ruiteix) [inline composer.json:5]: >=7.1

**Comment by ruiteix** [comment]
In case of bad body you have to return a 400 HTTP code.

**Comment by ruiteix** [comment]
Plz use `DECIMAL(10, 2)` type for price

**Comment by WizBhoo** [comment]
Thanks @ruiteix for your comments.
To answer your UML diagrams comments, please find bellow those diagrams modified.
All modifications are now implemented so if you are ok, the PR can be approved at your convenience.

4-2_Sequence_Gestion_Utilisateurs_POST_DELETE

7_BileMoAPIClassDiagram

8_MPD_BileMoAPI

**Pull request merged**